lang/c++/Makefile

```diff
@@ -1,4 +1,4 @@
-CC = g++
+CXX = g++
```

Comment:
using CXX seems to cause undefined reference errors. I tried CC and it seems to work fine.